AppTaskQueue/appscale/taskqueue/push_worker.py

```diff
@@ -39,6 +39,10 @@
 logger.setLevel(logging.INFO)
 
 
+def clean_task_name(task_name):
+  return task_name.decode('utf-8').replace("b'", "").replace("'", "")
```

Comment:
Please remove the replace calls. Even if they are a "just in case" thing, they have the potential to mask the underlying issue. If mangled names are being stored in the backend, we want it to fail as loudly as possible so that we can track down what is responsible for mangling the name in the first place.